src/common/get-all-users.js

```diff
@@ -3,3 +3,5 @@ export const getAllUsers = () => {
   const usersArr = JSON.parse(usersStr) || [];
   return usersArr;
 };
+
+export default getAllUsers;
```

Comment:
You already did a named export on line 1. Either remove the export keyword on line 1, or remove this line to use the named export.